src/components/search-box.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import { observer } from 'mobx-react'
+
+export default @observer class SearchBox extends Component {
+  onInputChange = event => {
+    this.props.artistsStore.searchedArtist = event.target.value
+  }
+
+  render() {
+    const { artistsStore } = this.props
+
+    return (
+      <div>
+        <div className="search-box__input-container">
+          <span className="search-box__artist-input-wrapper">
```

Comment:
Hmm tutaj też nie wiem czy to jest potrzebne? Czemu nie możesz połączyć `container` i `wrapper` w jeden element?

Reply:
uzywam tego, zeby wysrodkowac search boxa

Reply:
W `div` też możesz wyśrodkować. Wydaje mi się, że nie potrzebujesz tego dodatkowego `span` :P